hw/ip/aes/dv/aes_model_dpi/aes_model_dpi.c

```diff
@@ -129,11 +129,7 @@ void c_dpi_aes_crypt_message(unsigned char impl_i, unsigned char op_i,
   const unsigned char op = op_i & op_mask;
   const crypto_mode_t mode = (crypto_mode_t)(*mode_i & mode_mask);
   if (mode == kCryptoAesNone) {
-    printf(
         "ERROR: Mode kCryptoAesNone not supported by c_dpi_aes_crypt_message");
-    return;
-  }
-
   // key_len_i is one-hot encoded.
```

Comment:
This doesn't look right to me - doesn't this leave unbalanced parens?

Reply:
Closing this PR as @rswarbrick points out there are obvious issues here.
Sorry for not being more diligent